src/demo-app/input/input-container-demo.html

```diff
@@ -260,11 +260,11 @@
 <md-card class="demo-card demo-basic">
   <md-toolbar color="primary">Textarea Autosize</md-toolbar>
   <md-card-content>
-    <textarea md-autosize class="demo-textarea"></textarea>
+    <textarea md-textarea-autosize class="demo-textarea"></textarea>
```

Comment:
There should be a `mat` version too, but I've been lax on watching for that and will need to do another pass (probably at the same time I remove the deprecated selectors / properties).